modules/mod-deep-learning/ModelCardPanel.h

```diff
@@ -0,0 +1,139 @@
+
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+   Audacity(R) is copyright (c) 1999-2021 Audacity Team.
+
+   ModelCardPanel.h
+   Hugo Flores Garcia
+
+******************************************************************/
+/**
+
+\class ModelCardPanel
+\brief ModelCardPanel TODO
+
+*/
+/*******************************************************************/
+
+#pragma once
+
+#include "DeepModelManager.h"
+#include "ModelCard.h"
+#include "wx/colour.h"
+
+class DeepLearningEffectBase;
+class ShuttleGui;
+class ModelManagerPanel;
+
+class ModelCardPanel /* not final */: public wxPanelWrapper
```

Comment:
Might comment that this is an abstract class

Reply:
2161ece9c7f4b4533f82c7acaeaf375c1c9f3a48